Documentation/gettingstarted/encryption.rst

```diff
@@ -49,15 +49,17 @@ Enable Encryption in Cilium
 
 .. include:: k8s-install-download-release.rst
 
-Generate the required YAML files and deploy them:
+Deploy helm release:
 
-.. code:: bash
+.. parsed-literal::
+
+    helm install cilium/cilium --version |CHART_RELEASE| \\
```

Comment:
I think in principle, I'm reasonably happy with the `helm install` usage for getting started guides.

For encryption specifically though, given that the following sections are highlighting a bunch of other options you might use, it seems a little premature to launch straight into `helm install` right here. When I tried a command like this locally after already installing Cilium, I had the `helm` command complain at me that the resources already exist and helm doesn't want to override them.

Reply:
I have reviewed this section and it seems to be logically correct: we show that cilium can be installed via `helm install` and then highlight additional options. Consecutive `helm install` will fail since release is already in there, you will need `helm upgrade` to update values. This might be a bit confusing but I'm not sure hot to improve this section. Do you have any suggestions for that?

Reply:
Fair enough, I think if users are advanced enough that they're looking into transparent encryption then they should be able to figure out how to combine the helm options they need.